modules/hello-world/examples/hello_world_t.py

```diff
@@ -1,23 +1,41 @@
-"""
-[source,python]
-----
-"""
-# tag::connect[]
+from datetime import timedelta
+
 # needed for any cluster connection
-from couchbase.cluster import Cluster, ClusterOptions
 from couchbase.auth import PasswordAuthenticator
+from couchbase.cluster import Cluster
+# needed for options -- cluster, timeout, SQL++ (N1QL) query, etc.
+from couchbase.options import (ClusterOptions, ClusterTimeoutOptions,
+                               QueryOptions)
+
+# tag::connect[]
+# Update this to your cluster
+username = "username"
+password = "password"
+bucket_name = "travel-sample"
+cert_path = "path/to/certificate"
+# User Input ends here.
+
+# Connect options - authentication
+auth = PasswordAuthenticator(
+    username,
+    password,
+    # NOTE: If using SSL/TLS, add the certificate_path.
+    # We strongly reccomend this for production use.
+    # cert_path=cert_path
+)
 
-# needed to support SQL++ (N1QL) query
-from couchbase.cluster import QueryOptions
+# Connect options - global timeout opts
+timeout_opts = ClusterTimeoutOptions(kv_timeout=timedelta(seconds=10))
```

Comment:
We need this on Capella, where the assumption is the client is on a laptop many hops from AWS,
but not for Server, which is likely to be local Docker.

Reply:
(and WaitUntilReady is more useful for delays in spinning up the server and creating buckets locally)